internal/configuration.go

```diff
@@ -0,0 +1,17 @@
+package internal
+
+import "github.com/caarlos0/env/v11"
+
+type Configuration struct {
```

Comment:
Consider initializing your 'Configuration' struct with default values or use a method to verify after loading if the required fields are set, which can improve robustness.